Manager/AccessTokenManagerInterface.php

```diff
@@ -6,6 +6,9 @@
 
 use Trikoder\Bundle\OAuth2Bundle\Model\AccessToken;
 
+/**
+ * @method int clearRevoked()
```

Comment:
@franjo-zadelj-trikoder Add a note explaining that not defining this method is deprecated since version 3.2 and that the method will be added to the interface in v4.

Example from Symfony:

https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/HttpKernel/KernelInterface.php#L25

Reply:
and refresh and auth code manager interfaces should receive the same docblock. Currently you've put this only on the access token manager interface.